packages/core/src/model.ts

```diff
@@ -14,7 +14,7 @@ import {
 
 export function createModel<TContext, TEvent extends EventObject>(
   initialContext: TContext
-): Model<TContext, TEvent, any, any>;
+): Model<TContext, TEvent, void, void>;
```

Comment:
We shouldn't use `void` here; the behavior should be the same as `createMachine` without the model (any actions are supported)

Reply:
THis already got changed:
https://github.com/statelyai/xstate/blob/095e40652c3e83b440d1c0b546df64bafc263dd0/packages/core/src/model.ts#L21

Note that the last argument has to be `void` in this case. Otherwise those tests would not pass:
https://github.com/statelyai/xstate/blob/095e40652c3e83b440d1c0b546df64bafc263dd0/packages/core/test/model.test.ts#L478-L498